nni/retiarii/oneshot/pytorch/base_lightning.py

```diff
@@ -57,16 +73,11 @@ class BaseOneShotLightningModule(pl.LightningModule):
 
     Parameters
     ----------
-    base_model : pl.LightningModule
-        The evaluator in ``nni.retiarii.evaluator.lightning``. User defined model is wrapped by base_model, and base_model will
-        be wrapped by this model.
-    custom_replace_dict : Dict[Type[nn.Module], Callable[[nn.Module], nn.Module]], default = None
-        The custom xxxChoice replace method. Keys should be xxxChoice type and values should return an ``nn.module``. This custom
-        replace dict will override the default replace dict of each NAS method.
-    """
+    """ + _base_model_note + _custom_replace_dict_note
```

Comment:
this docstring looks hacky, why is it?

Reply:
Because I don't want to write super-class's docstring over and over again.
I learned this from PyTorch. They have quite a lot of docstrings like this. :)